code/modules/projectiles/gun_attachables.dm

```diff
@@ -166,9 +166,11 @@ Defined in conflicts.dm of the #defines folder.
 		// Apply bullet traits from attachment to gun's current projectile
 		G.in_chamber.apply_bullet_trait(L)
 
-/obj/item/attachable/proc/Detach(var/obj/item/weapon/gun/G)
+/obj/item/attachable/proc/Detach(var/mob/user, var/obj/item/weapon/gun/G)
```

Comment:
G -> detach_from or w/e u should've known this was gonna happen literally redefining the args......

Reply:
erm i hate oyu